application/app/connectors/dataverse/handlers/collections.rb

```diff
@@ -29,6 +29,13 @@ def show(request_params)
             success: false
           )
         end
+        collection_url = Dataverse::Concerns::DataverseUrlBuilder.build_collection_url(dataverse_url, @collection_id)
+        RepoRegistry.repo_history.add_repo(
+          collection_url,
+          ConnectorType::DATAVERSE,
+          title: collection.data.name,
+          note: 'collection'
+        )
```

Comment:
Not sure if it can make more sense to split `note` into 2 properties: `version` and `kind`. The latter can store the symbols `:collection` or `:dataset`. `version` can store the version when applicable.

Reply:
I consider that idea in one of the previous versions. I settled in the end for `note` so that each connector and each handler can choose what to use as a value. I selected `collection` is this case as couldn't find anything else worth showing.